adhocracy4/polls/models.py

```diff
@@ -148,6 +163,23 @@ class Choice(models.Model):
 
     objects = ChoiceQuerySet.as_manager()
 
+    def clean(self, *args, **kwargs):
+        if self.question.is_open:
+            raise ValidationError({
+                'question': _('Open questions cannot have choices.')
+            })
+        if self.is_other_choice and self.question.choices.count() == 0:
+            raise ValidationError({
+                'is_other_choice': _('"Other" cannot be the only choice. Use '
+                                     'open question or add another choice '
+                                     'first.')
```

Comment:
The text sounds a bit like you are only allowed to check the box if there are questions added first, so maybe without the first?
'"Other" cannot be the only choice. Use '
'open question or add more choices.'